docs/CONTRIBUTING.md

```diff
@@ -1,99 +1,16 @@
 # Contributing
-[Reatom](https://github.com/artalar/reatom) is open source state manager for small and complex applications. 
 
-If you want to contribute to improving the library, use the following instructions to create changes:
+**Hello friends!**
 
-- [Creating an issue](#creating-an-issue)
-- [Sending a Pull Request](#sending-a-pull-request)
+We are very happy if you are interested in the project. Would be grateful to you for any help in the development of Reatom.
 
-> **Note:** Languages other than English are not normally used in issue or commits descriptions.
+There are many ways to help the project:
 
-## Creating an issue
+- Add your star for [Reatom repository](https://github.com/artalar/reatom)
+- Write [issue](https://github.com/artalar/reatom/issues) about founds bugs with examples for reproduce
+- Proposal for [new features](https://github.com/artalar/reatom/issues) useful for the entire community
+- Write docs or fix current ([see instruction](https://github.com/artalar/reatom/blob/master/CONTRIBUTING.md))
+- Write code with new features or fix bugs ([see instruction](https://github.com/artalar/reatom/blob/master/CONTRIBUTING.md))
+- and more other ways...
```

Comment:
- Log any spotted issues or feature requests in [Reatom issue tracker](https://github.com/artalar/reatom/issues)
- Help with the development of new features or bug fixes ([see contribution guidelines](https://github.com/artalar/reatom/blob/master/CONTRIBUTING.md))
- Help with the documentation ([see contribution guidelines](https://github.com/artalar/reatom/blob/master/CONTRIBUTING.md))
- Spread the word about Reatom!
- Star [Reatom on GitHub](https://github.com/artalar/reatom)